modules/claims_api/spec/requests/v2/veterans/claims_request_spec.rb

```diff
@@ -954,6 +954,30 @@
         end
       end
 
+      describe 'participant ID' do
+        context 'when missing' do
+          let(:ccg_token) { OpenStruct.new(client_credentials_token?: true, payload: { 'scp' => [] }) }
+
+          it 'returns a 422' do
+            allow(JWT).to receive(:decode).and_return(nil)
+            allow(Token).to receive(:new).and_return(ccg_token)
+            allow_any_instance_of(TokenValidation::V2::Client).to receive(:token_valid?).and_return(true)
+
+            allow_any_instance_of(MPIData)
+              .to receive(:mvi_response).and_return(profile)
+
+            get claim_by_id_path, headers: { 'Authorization' => 'Bearer HelloWorld' }
+
+            expect(response.status).to eq(422)
+            json_response = JSON.parse(response.body)
+            expect(json_response['errors'][0]['detail']).to eq(
+              "Unable to locate Veteran's Participant ID in Master Person Index (MPI). " \
+              'Please submit an issue at ask.va.gov or call 1-800-MyVA411 (800-698-2411) for assistance.'
+            )
```

Comment:
these tests are validating the missing participant id. They're not testing the new item. You can tell by the missing `ICN` portion. It should have the below:
```
Veteran's ID/ICN in Master
```